CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 - **General:** Fail fast on `buildScalers` when not able to resolve a secret that a deployment is relying on ([#2394](https://github.com/kedacore/keda/pull/2394))
 - **General:** `keda-operator` Cluster Role: add `list` and `watch` access to service accounts ([#2406](https://github.com/kedacore/keda/pull/2406))|([#2410](https://github.com/kedacore/keda/pull/2410))
 - **General:** Sign KEDA images published on GitHub Container Registry ([#2501](https://github.com/kedacore/keda/pull/2501))|([#2502](https://github.com/kedacore/keda/pull/2502))|([#2504](https://github.com/kedacore/keda/pull/2504))
+- **General:** Updated HTTPClient to be proxy-aware, if available, from environment variables. ([#2577](https://github.com/kedacore/keda/issues/2577))
```

Comment:
Please move this to Unreleased section of the changelog, 2.6 is already out :)

Reply:
wow, nice catch!
Probably it's better if you rebase main also because you branch is more than 70 commits behind `main` (it's not mandatory but probably you will need to do it to avoid possible conflicts)